controllers/index.controller.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../db/index');
+
+exports.showHomePage = async (req, res, next) => {
+  try {
+    res.render('home');
+  } catch (e) {
```

Comment:
@kayheen can try query the right data in here and pass it into the view to be rendered.

Reply:
got it! attempted in #11